app/gui/integration-test/project-view/collapsingAndEntering.spec.ts

```diff
@@ -303,6 +398,8 @@ async function expectInsideMain(page: Page) {
 }
 
 async function expectInsideFunc1(page: Page) {
+  // The mouse is often in output port area, making our checks fooled by the edge ghost.
```

Comment:
Maybe we should test the number of connected edges, so it isn't affected by mouse position

Reply:
I mean we shouldn’t have any unconnected edges either… Perhaps it would be a hilarious bug to face, but I don’t see much benefit in counting connected edges only.